client/src/controllers/TabsController.ts

```diff
@@ -0,0 +1,280 @@
+import { Controller } from '@hotwired/stimulus';
+
+const TAB_DEFAULTS = {
+  css: {
+    animate: 'animate-in',
+  },
+  keys: {
+    end: 'End',
+    home: 'Home',
+    left: 'ArrowLeft',
+    up: 'ArrowUp',
+    right: 'ArrowRight',
+    down: 'ArrowDown',
+  },
+  direction: {
+    ArrowLeft: -1,
+    ArrowRight: 1,
+  },
+}
+
```

Comment:
I'd like to see us try to use the Stimulus keyboard actions instead of having these defaults set in the JS.